src/tray/vars/all/main.yml

```diff
@@ -14,12 +14,10 @@ peertube_runners:
       - vod-web-video-transcoding
       - vod-hls-transcoding
       - vod-audio-merge-transcoding
-
-
-peertube_runner_app_resources:
-  requests:
-    cpu: 50m
-    memory: 500Mi
-  limits:
-    cpu: 3
-    memory: 1Gi
+    app_resources:
+      requests:
+        cpu: 3
+        memory: 1.5Gi
+      limits:
+        cpu: 3.2
```

Comment:
FMI, is it the load average max ?

Reply:
Also interested to know how you picked these values @lunika

Reply:
I monitor the runner in production to determine these values.
It is the CPU limit the runner can consume. IMO it will consume the most it has. For the memory it's the same purpose. After some research you can not limit ffmpeg resource consumption. It will use wll the resources it can.

https://stackoverflow.com/questions/12390682/how-can-i-limit-ffmpeg-memory-usage

https://superuser.com/questions/1570417/is-there-any-way-to-limit-ffmpeg-s-memory-usage